suite-common/suite-utils/src/device.ts

```diff
@@ -442,3 +450,6 @@ export const isDeviceWithButtons = (
     deviceModel: DeviceModelInternal,
 ): deviceModel is DeviceModelInternal.T1B1 | DeviceModelInternal.T2B1 | DeviceModelInternal.T3B1 =>
     deviceModel.at(2) === 'B';
+
+export const isAnyDeviceEventAction = (action: AnyAction): action is DeviceEvent =>
+    Object.values(DEVICE).includes(action.type as DeviceEvent['type']);
```

Comment:
Nit: get rid of the `as`
```suggestion
    isArrayMember(action.type, Object.values(DEVICE));
```